### ISSUE TITLE
Code editing in Human Review phase

### ISSUE THREAD

**Issue by william2313** [OpenBMB/ChatDev#95]
I made some changes to the generated code myself during the Human review phase. I then explained this in the Human-Agent-Interaction and asked for some other changes. However, in the next iteration, the code had reverted to the previous version from ChatDev, ignoring my changes. 

Is there a way to edit code files so that this is used by ChatDev in the next iteration? This would be extremely useful, as in practice it is often easier and more effective to fix bugs yourself than asking ChatDev to do it.

**Comment by Dartomic** [comment]
Did the chatbots acknowledge your alterations and requests for other changes?

**Comment by william2313** [comment]
Not really. I saw my review text repeated a few times in the log, but they didn't refer to it in any of their own talk. The code that is shown in the log right after my request is still the old code. Clearly it has it in memory and doesn't reload it from the files. (I had asked them specifically NOT to change my edits lol).

**Comment by Dartomic** [comment]
I guess their becoming more human every day. lol
